agents/windows/plugins/mk_oracle.ps1

```diff
@@ -146,7 +146,11 @@ else {
      debug_echo "${CONFIG_FILE} not found"
 }
 
-
+# If sqlnet.ora or tnsnames.ora exist in MK_CONFDIR set it as %TNS_ADMIN%
+if ((test-path -path "${MK_CONFDIR}\sqlnet.ora") -or (test-path -path "${MK_CONFDIR}\tnsnames.ora")) {
+     $env:TNS_ADMIN = "${MK_CONFDIR}"
+     debug_echo "value of TNS_ADMIN = $env:TNS_ADMIN"
+}
```

Comment:
Just one little thing: There is an else missing - you need to call the debug echo also if these files are not located in MK_CONFDIR.

Reply:
Changed the debug_echo to always be called. And removed a quote which caused the oracle_memory sql statement to fail.

ORCL779|SGA_info|Free SGA Memory Available|0
ERROR:
ORA-01740: missing double quote in identifier